extensions/positron-r/package.nls.json

```diff
@@ -38,6 +38,7 @@
 	"r.configuration.interpreters.exclude.markdownDescription": "List of absolute paths to R binaries or folders containing R installations to exclude from the available R interpreters. These interpreters will not be displayed in the Positron UI.\n\nIf an interpreter is both included via `#positron.r.customRootFolders#` or `#positron.r.customBinaries#` and excluded via `#positron.r.interpreters.exclude#`, it will not be displayed in the Positron UI. See also `#positron.r.interpreters.override#`.\n\nExample: On Linux, add `/custom/location/R/4.3.0/bin/R` to exclude the specific binary, or `/custom/location` to exclude any R installations within the directory.\n\nRequires a restart to take effect.",
 	"r.configuration.interpreters.override.markdownDescription": "List of absolute paths to R binaries or folders containing R installations to override the list of available R interpreters. Only the interpreters found at the specified paths will be displayed in the Positron UI.\n\nThis setting takes precedence over the `#positron.r.customBinaries#`, `#positron.r.customRootFolders#`, and `#positron.r.interpreters.exclude#` settings.\n\nExample: On Linux or Mac, add `/custom/location/R/4.3.0/bin/R` to include only this specific installation, or `/custom/location` to include only R installations found within the directory.\n\nRequires a restart to take effect.",
 	"r.configuration.interpreters.default.markdownDescription": "Absolute path to the default R binary to use for new workspaces. This setting no longer applies once you select an R interpreter for the workspace.\n\nExample: On Linux, add `/custom/location/R/4.3.0/bin/R` to set the default interpreter to the specific R binary.\n\nRequires a restart to take effect.",
+	"r.configuration.interpreters.condaDiscovery.markdownDescription": "Enable discovery of R installations within conda/mamba environments. **Experimental**: Support for conda environments in Positron is experimental and may not work correctly in all cases. When enabled, R installations found in conda environments will be available for selection, and the conda environment will be automatically activated when starting R to ensure compilation tools and dependencies are available.\n\nRequires a restart to take effect.",
```

Comment:
```suggestion
	"r.configuration.interpreters.condaDiscovery.markdownDescription": "Enable discovery of R installations within conda/mamba environments. Support for conda environments in Positron is experimental and may not work correctly in all cases. When enabled, R installations found in conda environments will be available for selection, and the conda environment will be automatically activated when starting R to ensure compilation tools and dependencies are available.\n\nRequires a restart to take effect.",
```

So that the setting doesn't have "experimental" quite so many times:

<img width="834" height="437" alt="Image" src="https://github.com/user-attachments/assets/65283f04-2ed1-48a5-8154-63fd182127dd" />